x/protorev/genesis.go

```diff
@@ -17,6 +17,15 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	// Init module parameters
 	k.SetParams(ctx, genState.Params)
 
+	// Init module state
+	k.SetProtoRevEnabled(ctx, genState.Params.Enabled)
+	k.SetDaysSinceGenesis(ctx, 0)
```

Comment:
This name going to confuse future developers I assume, similar to how @ValarDragon was confused. Perhaps we rename it to something that indicates we mean since Protorev Genesis (e.g. k.SetDaysSinceModuleGenesis or k.SetDaysSinceProtorevGenesis)